pandas/io/formats/style.py

```diff
@@ -1222,6 +1222,35 @@ class MyStyler(cls):
 
         return MyStyler
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, *args, **kwargs)
```

Comment:
This should be a little more descriptive.  Apply `func` to what?  Also, for what purpose?

(yes, as reviewers, we can see what the rationale was, so just put that rationale into the docstring)